catalog/USFS/USFS_GTAC_TreeMap_v2016.jsonnet

```diff
@@ -0,0 +1,2351 @@
+
+    TreeMap v2016 contains one image, a 22-band 30 x 30m resolution gridded map of the forests of the continental United States circa 2016, with each band 
+    representing an attribute derived from select FIA data (and one band representing the TreeMap ID). Examples of attributes include forest type, canopy 
+    cover percent, live tree stocking, live/dead tree biomass, and carbon in live/dead trees.
+    
+    The 2016 methodology includes disturbance as a response variable, resulting in increased accuracy in mapping disturbed areas. Within-class accuracy was
+    over 90% for forest cover, height, vegetation group, and disturbance code when compared to LANDFIRE maps. At least one pixel within the radius of
+    validation plots matched the class of predicted values in 57.5% of cases for forest cover, 80.0% for height, 80.0% for tree species with highest basal
+    area, and 87.4% for disturbance.
+
+    **Additional Resources**
+
+    * Please see the [TreeMap 2016 Publication](https://www.fs.usda.gov/research/treesearch/65597) for more detailed information regarding methods and accuracy assessment.
+
+    * The [TreeMap 2016 Data Explorer](https://apps.fs.usda.gov/lcms-viewer/treemap.html) is a web-based application that
+      provides users the ability to view TreeMap attribute data.
+
+    * The [TreeMap Research Data Archive](https://www.fs.usda.gov/rds/archive/Catalog/RDS-2021-0074) for the full dataset download, metadata,
+      and support documents.
+
+    * [TreeMap Raster Data Gateway](https://data.fs.usda.gov/geodata/rastergateway/treemap/) for separated attribute data downloads, metadata, and support documents.
+
+    * [FIA Database Manual version 8](https://www.fia.fs.usda.gov/library/database-documentation/current/ver80/FIADB%20User%20Guide%20P2_8-0.pdf)
+      for more detailed information on the attributes included in TreeMap 2016.
+
+    Contact [sm.fs.gtactreemap@usda.gov](mailto:sm.fs.gtactreemap@usda.gov) with any
+    questions or specific data requests.
+
+    * **Forest Inventory Analysis. 2022a.**
+    Forest Inventory Analysis DataMart.
+    Forest Inventory Analysis DataMart FIADB_1.9.0. 2022. https://apps.fs.usda.gov/fia/datamart/datamart.html.
+
+    * **Ohmann, Janet L and Matthew J Gregory. 2002.**
+    Predictive Mapping of Forest Composition and Structure with Direct Gradient Analysis and Nearest- Neighbor Imputation in Coastal Oregon, USA. 
+    Can. J. For. Res. 32:725-741. [doi: 10.1139/X02-011](https://doi.org/10.1139/X02-011).
+
+    * **Pierce, Kenneth B Jr, Janet L Ohmann, Michael C Wimberly, Matthew J Gregory, and Jeremy S Fried. 2009.**
+    Mapping Wildland Fuels and Forest Structure for Land Management: A Comparison of Nearest Neighbor Imputation and Other Methods.
+    Can. J. For. Res. 39: 1901-1916. [doi:10.1139/X09-102](https://doi.org/10.1139/X09-102).
+
+    * **Riley, Karin L.; Grenfell, Isaac C.; Finney, Mark A.; Shaw, John D. 2021.** 
```

Comment:
This can be moved to a sci:publications field, as it's directly relevant to the data. 

example: https://github.com/google/earthengine-catalog/blob/main/catalog/CAS/CAS_IGSNRR_PML_V2_v017.jsonnet#L173

Oh wait, is this is the same as the sci:citation field? Then we don't need to repeat it.